teacher.rb

```diff
@@ -0,0 +1,11 @@
+require './person'
+class Teacher < Person
+  def initialize(age, name = 'unknown', parent_permision: true)
+    super(age, name, parent_permision)
+    @specialization = specialization
+  end
```

Comment:
- [ ] specialization should be included in the arguments of the initialize method as well